lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -274,18 +278,71 @@ def test_course_end_date_after_course(self):
         )
         self.assertEqual(block.title, 'Course End')
 
-    ## Tests Verified Upgrade Deadline Date Block
+    # Tests Verified Upgrade Deadline Date Block
+
+    def check_upgrade_banner(self, banner_expected=True, include_url_parameter=True):
+        """
+        Helper method to check for the presence of the Upgrade Banner
+        """
+        url = reverse('info', args=[self.course.id.to_deprecated_string()])
+        if include_url_parameter:
+            url += '?upgrade=true'
+        resp = self.client.get(url)
+        if banner_expected:
```

Comment:
Nit: define the expected string value as a variable. That way if it changes, we are sure it changes in both usages (since it is easy to get bogus "not in" tests with incorrect strings).